src/common/Header.jsx

```diff
@@ -15,49 +25,99 @@ export const hideHeaderFooter = (path) => {
               {links.map(link => {
                 let isActive = pathname.match(new RegExp('^' + link.href))
                 if(link.href === '/') isActive = link.href === pathname
 
                 return (
-                  <li key={link.name}>
-                    <a
-                      href={link.href}
-                      className={
-                          isActive
-                          ? 'nav-link active'
-                          : 'nav-link'
-                      }
-                      target={link.name === 'Filing' ? '_blank' : null}
-                      rel={
-                        link.name === 'Filing' ? 'noopener noreferrer' : null
-                      }
-                    >
-                      {link.name}
-                    </a>
+                  <li key={link.name} className="usa-nav__primary-item">
+                    {!link.submenu ? 
+                      <Link
+                        to={link.href}
+                        className={
+                            isActive
+                            ? 'usa-nav__link usa-current'
+                            : 'usa-nav__link'
+                        }
+                        //target={link.name === 'Filing' ? '_blank' : null}
+                        rel={
+                          link.name === 'Filing' ? 'noopener noreferrer' : null
+                        }
+                      >
+                        {link.name}
+                      </Link> :
+                      <>
+                        <button
+                          type="button"
+                          className="usa-accordion__button usa-nav__link"
+                          aria-expanded="false"
+                          aria-controls={link.name}
+                        >
+                          <span>{link.name}</span>
+                        </button>
+                        <ul id={link.name} className="usa-nav__submenu">
+                          {link.submenu.map(sublink => (<li key={sublink.name} className="usa-nav__submenu-item">
+                            {sublink.name == "Developer APIs" || sublink.name == "FAQs" ?
+                              <Link
+                                to={sublink.href}
+                                onClick={() => forceReload(sublink.href)}
+                              >
+                                {sublink.name}
+                              </Link> :
+                              !sublink.href ? 
+                                <div className="subMenuHeading">{sublink.name}</div> :
+                                <Link
+                                  to={sublink.href}
+                                >
+                                  {sublink.name}
+                                </Link>
+                            }
+                          </li>)
+                          )}
+                        </ul>
+                      </>
+                    }
```

Comment:
This feels a bit dense with all the ternary operators.  Consider breaking things out into a few sub-components.

Reply:
Going to leave this as is for now. Can come back and refactor later.